### ISSUE TITLE
[1.8.9] Angel ring breaks gamemode flight

### ISSUE THREAD

**Issue by TATDK** [rwtema/extrautilities_old_issues#849]
If a player in spectator mode have an angel ring that is out of power, the player falls into the void and dies.

Version: extrautils2-1.8.9-alpha-0.0.2

**Comment by hron84** [comment]
In gamemode, you mean creative mode flight? You used angel ring in creative?

**Comment by TATDK** [comment]
Yes, creative flight. Though in spectator mode you have no-clip and therefore falls right through all blocks.

I'm a moderator on the server and goes into different gamemodes from time to time and doesn't always dump my inventory into a chest.

**Comment by hron84** [comment]
You get me wrong, I didn't wanted any negative meaning for my question, just wanted to clarify things. Thanks for the update.

**Comment by KnightMiner** [comment]
Does it also prevent flight in creative mode? I know you won't fall through the world then, but it would be related.

**Comment by TATDK** [comment]
@hron84 Sry, if my message looked like I took it with any negative meaning. I answered when I was tired and about to go to bed. No negative meaning taken ^w^

@KnightMiner No, it does not negate flight in creative mode.
